providers/gcp/resources/discovery/gcr/gcr.go

```diff
@@ -112,13 +110,8 @@ func (a *GcrImages) ListRepository(repository string, recursive bool) ([]*asset.
 }
 
 // List uses your GCP credentials to iterate over all your projects to identify potential repos
-func (a *GcrImages) List() ([]*asset.Asset, error) {
-	assets := []*asset.Asset{}
-	// repoAssets, err := a.ListRepository("index.docker.io/mondoo/client", false)
-	// if err == nil && repoAssets != nil {
-	// 	assets = append(assets, repoAssets...)
-	// }
-	// return assets, nil
+func (a *GcrImages) List() ([]*inventory.Asset, error) {
+	assets := []*inventory.Asset{}
 
 	resSrv, err := cloudresourcemanager.NewService(context.Background())
```

Comment:
Can we do this via MQL?

Reply:
Perhaps...

But this would require adding multiple MQL resources. And I'd prefer to actually get rid of the repository resource we currently have: https://github.com/mondoohq/cnquery/issues/1651

This command is only about the images. We do not care about the rest at this point.

Reply:
As @vjeffrey had the same question, I dug a bit deeper. Now, I think it isn't possible: https://github.com/mondoohq/cnquery/pull/1650#discussion_r1319335451